reference_entities/create_new_attribute_type.rst

```diff
@@ -0,0 +1,35 @@
+Create a new Reference Entity Attribute type
+============================================
+
+.. note::
+
+   Reference Entities feature is only available for the **Enterprise Edition**.
+
+This cookbook will present how to create your own Attribute Type for Reference Entities.
+Currently, there are 6 types of Attribute for Reference Entities:
+
+- Text
+- Image
+- Record (*Link Record to another Record*)
+- Record Collection (*Link Record to several Records*)
+- Option
+- Option Collection
+
+Requirements
+------------
+
+During this chapter, we assume that you already created a new bundle to add your custom Reference Entity Attribute. Let's assume its namespace is ``Acme\\CustomBundle``.
+
+Create the Attribute
```

Comment:
I would rewrite those two last section as follow, without giving too much details about the how.
```
To make the PIM support a custom attribute, there are two main steps:
1. Create the attribute model and the services that will let us create the attribute, and edit its properties.
2. Create the record's value related to this new attribute. 
```